src/main/java/com/ibanity/apis/client/products/xs2a/services/impl/TransactionsServiceImpl.java

```diff
@@ -37,7 +38,7 @@ public TransactionsServiceImpl(ApiUrlProvider apiUrlProvider, IbanityHttpClient
             pagingSpec = IbanityPagingSpec.DEFAULT_PAGING_SPEC;
         }
 
-        String url = getUrl(transactionsReadQuery.getFinancialInstitutionId(), transactionsReadQuery.getAccountId());
+        String url = getUrl(transactionsReadQuery.getFinancialInstitutionId(), transactionsReadQuery.getAccountId(), transactionsReadQuery.getSynchronizationId());
```

Comment:
Maybe you could add a test in TransactionsServiceImplTest -> @Test listUpdatedTransactions, in order to make sure we don't break this behavior in the future?

Reply:
added